collections/King's College Cambridge/Kings_1.xml

```diff
@@ -45,7 +45,7 @@
 				<msContents>
 					<summary> 1 copy of <title>Irshād al-Sālikīn</title>.</summary> 									
 				<msItem xml:id="Kings_1-item1" n="1"> 
-					<author key="person_f5738">
+					<author key="person_f5738" >
```

Comment:
This change seems strange to me -- generally there is no space after the final attribute value and before the closing `>` in XML. It's also not this way in any other tag (see the `msItem` example just above it.)